CarRecognition/Source Files/Modules/Recognition/Root/RecognitionViewController.swift

```diff
@@ -112,16 +112,19 @@ internal final class RecognitionViewController: TypedViewController<RecognitionV
         augmentedRealityViewController.updateDetectionViewfinder(to: mostConfidentRecognition, normalizedConfidence: normalizedConfidence)
         switch mostConfidentRecognition.recognition {
         case .car(let car):
+            classificationService.set(state: .paused)
+            inputNormalizationService.reset()
+            addSlidingCard(with: car)
+            carsDataService.mark(car: car, asDiscovered: true)
+            
             if normalizedConfidence >= arConfig.neededConfidenceToPinLabel {
-                augmentedRealityViewController.addPin(to: car, completion: { [unowned self] car in
-                    self.classificationService.set(state: .paused)
-                    self.inputNormalizationService.reset()
-                    self.addSlidingCard(with: car)
-                    self.carsDataService.mark(car: car, asDiscovered: true)
-                }, error: { [unowned self] error in
+                augmentedRealityViewController.addPin(to: car, completion: nil) { [unowned self] error in
                     // TODO: Debug information, remove from final version
```

Comment:
Please remove those two TODOd, I think it's not necessary anymore (this labels are hidden in production env).
Instead of TODO please leave comment like this:
```
// Debug information, invisible in prodution
```